testing/kea/APKBUILD

```diff
@@ -0,0 +1,174 @@
+# Maintainer: Steve Holweg <skytep@gmail.com>
+# Contributor: Steve Holweg <skytep@gmail.com>
+# Contributor: Baptiste Jonglez <baptiste--aur at jonglez dot org>
+# Contributor: nfnty
+
+pkgname=kea
+pkgver=1.3.0
+pkgrel=0
+pkgdesc="High-performance, extensible DHCP server engine from ISC, supporting both DHCPv4 and DHCPv6"
+arch="all"
+url="http://kea.isc.org"
+license="MPL2"
+depends=""
+makedepends="botan log4cplus boost-dev postgresql-dev mariadb-dev ccache"
+subpackages="$pkgname-doc $pkgname-dev $pkgname-admin:admin:noarch $pkgname-ctrl-agent:ctrlagent $pkgname-dhcp-ddns:dhcpddns $pkgname-dhcp4 $pkgname-dhcp6 $pkgname-keactrl:keactrl:noarch $pkgname-utils"
+
+source="https://ftp.isc.org/isc/kea/${pkgver}/kea-${pkgver}.tar.gz
+        fix-scripts-include-path.patch
+        kea-dhcp4.service
+        kea-dhcp6.service
+        kea-dhcp-ddns.service
+        kea-ctrl-agent.service"
+
+validpgpkeys="BE0E9748B718253A28BB89FFF1B11BF05CF02E57" # Internet Systems Consortium, Inc. (Signing key, 2017-2018) <codesign@isc.org>
+
+prepare() {
+  cd "$builddir"
+  # The include path is set incorrectly (it uses undefined $prefix instead of /usr).
+  # Also, it fallbacks to the absolute path inside the chrooted build dir,
+  # which will certainly not exist on the target system...
+  patch -p0 < "${srcdir}"/fix-scripts-include-path.patch
```

Comment:
patch should be modified to `-p1` instead of `-p0` and then can the entire prepare funtion be removed. The comment should be included in the patch itself.